### DISCUSSION TITLE
build with jekyll failed on github pages after a few deployments

### DISCUSSION THREAD

**Discussion by Aline89b** [community/community#35708] [Other Feature Feedback, Questions, & Ideas]
Hello all!

I'm studying fe development and im trying to deploy a project. It worked ok til the last deployment with wepack, using gh-pages but today it gave me this error 

 Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    No such file or directory @ dir_chdir - /github/workspace/docs

I don't know where to look for the issue.. I used only simple css, no sass or others.

https://github.com/Aline89b/library here is the repo.
thank you ,
if someone can help would be nice 
Aline

**Comment by airtower-luna** [comment]
It looks like you're trying to use the default Jekyll deployment for something that isn't a Jekyll site, so Jekyll fails to build it.

I'm not sure exactly how you build your site, but it seems like a Node-based build at a glance. In that case you might want to set up a custom deployment workflow that runs whatever build steps you need, see: https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow
